src/connectionmanager.cpp

```diff
@@ -813,7 +813,7 @@ void ConnectionManager::statusUpdated(const QString& serverIpv4Gateway,
 
   list.swap(m_getStatusCallbacks);
   for (const std::function<void(
-           const QString& serverIpv4Gateway, const QString& deviceIpv4Address,
+           const QString&serverIpv4Gateway, const QString&deviceIpv4Address,
```

Comment:
I'm surprised this linter issue is coming back up again since we updated clang-format a while ago.